### ISSUE TITLE
K8s/Deployment: Monitor deployment

### ISSUE THREAD

**Issue by caoxianfei1** [opencurve/curve-operator#26]
Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)

Currently, the Curve-Operator does not have access to monitoring, but the monitoring indicators and data of the cluster are very necessary.

Describe the solution you'd like (描述你期望的解决方法)

You can try to introduce prometheus and Grafana to monitor the health of Curve cluster. For convenience, you may not need to deploy bare prometheus and grafana, you can directly introduce some mature solutions in the CNCF community, such as prometheus-operator etc.

curve monitor deployment steps now :[部署 CurveBS 监控](https://github.com/opencurve/curveadm/wiki/curvebs-monitor-deployment)
curve-operator refer to repository: https://github.com/opencurve/curve-operator
Prometheus-operator refer to repository: https://github.com/prometheus-operator/prometheus-operator

**Comment by zhanghuidinah** [comment]
- **This issue has been chosen for community developer activities.  Welcome to feel free to try it. After the PR is merged, you will receive a gift package and a certificate as a community contributor.**

- **Developer Activities Registration Steps please ref https://github.com/opencurve/curve/issues/2334#issuecomment-1504723771**

**Comment by Gala-R** [comment]
i want to try it.

**Comment by caoxianfei1** [comment]
@Gala-R You can contact us anytime if you have any questions about monitor. Firstly you maybe need to learn how to deploy prometheus and grafana not in kubernetes enviropnment. here : https://github.com/opencurve/curveadm/wiki/curvebs-monitor-deployment

**Comment by Gala-R** [comment]
> @Gala-R You can contact us anytime if you have any questions about monitor. Firstly you maybe need to learn how to deploy prometheus and grafana not in kubernetes enviropnment. here : https://github.com/opencurve/curveadm/wiki/curvebs-monitor-deployment

Thank you for your apply, I will contact you if there is any problem.

**Comment by caoxianfei1** [comment]
@Gala-R How is going on?

**Comment by Gala-R** [comment]
> @Gala-R How is going on?

Sorry for being too busy at work recently, I will submit a PR during the May Day holiday

**Comment by Gala-R** [comment]
Can curve-operator only be deployed on Kubernetes 1.19, 1.20? The Kubernetes version is too old

**Comment by Gala-R** [comment]
```
2023-05-01 09:31:25.194733 I | chunkserver: node=172.17.0.10	device=/dev/vdb	formatted=82/80	status=Done
2023-05-01 09:31:25.196002 I | chunkserver: node=172.17.0.3	device=/dev/vdb	formatted=81/80	status=Done
2023-05-01 09:31:25.196008 I | chunkserver: node=172.17.0.7	device=/dev/vdb	formatted=82/80	status=Done
2023-05-01 09:31:43.779745 I | chunkserver: all format jobs has finished.
2023-05-01 09:31:43.813816 I | chunkserver: all jobs run completed in 24 hours
2023-05-01 09:31:43.857278 I | topology: {"servers":[{"name":"172.17.0.10_0","internalip":"172.17.0.10","internalport":8200,"externalip":"172.17.0.10","externalport":8200,"zone":"zone1","physicalpool":"pool1"},{"name":"172.17.0.3_0","internalip":"172.17.0.3","internalport":8200,"externalip":"172.17.0.3","externalport":8200,"zone":"zone2","physicalpool":"pool1"},{"name":"172.17.0.7_0","internalip":"172.17.0.7","internalport":8200,"externalip":"172.17.0.7","externalport":8200,"zone":"zone3","physicalpool":"pool1"}],"logicalpools":[{"name":"pool1","replicasnum":3,"zonenum":3,"copysetnum":1,"type":0,"scatterwidth":0,"physicalpool":"pool1"}],"npools":1}
2023-05-01 09:31:43.911134 I | topology: job created to generate physical_pool
2023-05-01 09:31:43.911157 I | chunkserver: create physical pool successed
2023-05-01 09:31:44.079380 E | chunkserver: go routinue exit because check time is more than 24 hours
2023-05-01T09:31:44.099Z	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "curvecluster", "request": "curve/my-cluster"}
2023-05-01T09:31:44.100Z	INFO	controllers.CurveCluster	reconcileing CurveCluster	{"curvecluster": "curve/my-cluster"}
2023-05-01 09:31:44.100322 I | controller: Cluster has been exist but need configured but we don't apply it now, you need delete it and recreate it!!!<->namespace="curve"
2023-05-01T09:31:44.149Z	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "curvecluster", "request": "curve/my-cluster"}
```
When I deployed the Curve BS cluster according to the steps in the README, I found that the chunkserver pod was not created, and this message appeared in the operator log

**Comment by caoxianfei1** [comment]
The bug has been fixed and you can deploy it again

**Comment by caoxianfei1** [comment]
> Can curve-operator only be deployed on Kubernetes 1.19, 1.20? The Kubernetes version is too old

yes, the apiextensions.k8s.io/v1beta1 group has been deprecated and some of the content is not validated.  upgrading the k8s version my be considerd In the future.

**Comment by caoxianfei1** [comment]
@Gala-R go on?

**Comment by MateSousa** [comment]
@caoxianfei1 Can i try if @Gala-R not available?
